cypress/e2e/patient_spec/encouter-notes.cy.ts

```diff
@@ -0,0 +1,78 @@
+import { FacilityCreation } from "@/pageObject/facility/FacilityCreation";
+import { PatientEncounter } from "@/pageObject/Patients/PatientEncounter";
+
+const facilityCreation = new FacilityCreation();
+const patientEncounter = new PatientEncounter();
+
+describe("Encounter Notes - Threads, Messages, and Notes", () => {
+  beforeEach(() => {
+    cy.loginByApi("devdoctor"); // logging in as doctor
+    cy.visit("/");
+    facilityCreation.selectFacility("GHC Trikaripur");
+  });
+
+  it("create multiple threads and add messages", () => {
+    patientEncounter
+      .navigateToEncounters()
+      .openFirstEncounterDetails()
+      .openNotesTab()
+      
+
+    // first Thread
+    cy.contains("button", "New Thread").click();
```

Comment:
```suggestion
    cy.contains("button", "New Thread").click();
```

as the per given guidelines, use the command function, cy.verifyandclickelement() to click on any button with text. Make sure you uses a proper data-cy test id's or element id ,